demos/demo33_test_wang.py

```diff
@@ -67,10 +67,10 @@ def logpi_like(x, grad=False):
     misfit = y_data - forward_eval
     if grad:
         direction = np.array([prec_obs * misfit])
-        grad_eval = gradient(direction=direction, wrt=x)
+        grad_eval = gradient(direction=direction, x=x)  # Use x=x instead of wrt=x
```

Comment:
```suggestion
        grad_eval = gradient(direction=direction, x=x) 
```
Also here I think no need to keep history of interface, that we used to call this wrt. I see demos timeless and relevant to most recent version of CUQIpy